### PR TITLE
fix: forgot password width

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Should be fixed as shown below

Screenshot 2023-09-12 at 9 14 29 PM
 now than just 

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by capJavert — APPROVED** [review]

File: packages/shared/src/components/auth/ChangePasswordForm.tsx [hunk 1]
```diff
@@ -94,6 +94,7 @@ function ChangePasswordForm({
           inputId="password"
           type="password"
           name="password"
+          className={{ container: 'w-full' }}
```
Comment (rebelchris) [inline packages/shared/src/components/auth/ChangePasswordForm.tsx:97]: Do we have this anywhere else in the code?
Wondering if we need to apply it in more places maybe

**Review by rebelchris — APPROVED** [review]

File: packages/shared/src/components/auth/ChangePasswordForm.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/auth/ChangePasswordForm.tsx:97]: Let me verify quickly.

Comment (sshanzel) [reply packages/shared/src/components/auth/ChangePasswordForm.tsx:97]: Last form we haven't seen. Seems to be working as intended 🚀 

Screenshot 2023-09-12 at 9 39 21 PM

Comment (rebelchris) [reply packages/shared/src/components/auth/ChangePasswordForm.tsx:97]: Alright thanks for checking, good to go

**Pull request merged**